packages/recorder/plugins/buffer/buffer.js

```diff
@@ -0,0 +1,234 @@
+const AudioContext =
+  window.AudioContext || /* istanbul ignore next */ window.webkitAudioContext;
+let audioContext;
+
+const BYTES_PER_SAMPLE = 2;
+
+/**
+ * Encode a Float32Array to an Uint8Array. Normalizes the audio.
```

Comment:
normalizing audio is bringing the amplitude to one target level. Why are we doing that here?